test/WebWorkerMainTemplatePlugin.test.js

```diff
@@ -1,13 +1,15 @@
-var should = require("should");
-var sinon = require("sinon");
-var WebWorkerMainTemplatePlugin = require("../lib/webworker/WebWorkerMainTemplatePlugin");
-var applyPluginWithOptions = require("./helpers/applyPluginWithOptions");
+"use strict";
+
+const should = require("should");
+const sinon = require("sinon");
+const WebWorkerMainTemplatePlugin = require("../lib/webworker/WebWorkerMainTemplatePlugin");
+const applyPluginWithOptions = require("./helpers/applyPluginWithOptions");
 
 describe("WebWorkerMainTemplatePlugin", function() {
-	var env;
+	let env;
 
-	beforeEach(function() {
-		env = {};
+	beforeEach(() => {
+		env = {} 
```

Comment:
tried to replace it with complete arrow function removing `{` and then decided against it. The effect of that :disappointed: